src/main/java/frc/robot/motion_profiling/AutoPath.java

```diff
@@ -6,9 +6,9 @@
  * This enum represent and hold the instances of auto paths
  */
 public enum AutoPath {
-    ExamplePath(new Path(new Waypoint(0, 0, 0), new Waypoint(fieldConstants.feederConstants.SIDE_WALL_TO_MIDDLE_FEEDER,
-        fieldConstants.feederConstants.ROCKET_TO_FEEDER, 90))), // this example uses field constants
-    ExampleReversePath(new Path(true, new Waypoint(0, 0, 0), new Waypoint(3, 2, 90))),
+  //  ExamplePath(new Path(new Waypoint(0, 0, 0), new Waypoint(fieldConstants.feederConstants.SIDE_WALL_TO_MIDDLE_FEEDER,
+   //     fieldConstants.feederConstants.ROCKET_TO_FEEDER, 90))), // this example uses field constants
+  //  ExampleReversePath(new Path(true, new Waypoint(0, 0, 0), new Waypoint(3, 2, 90))),
```

Comment:
Remove outdated examples instead of commenting them
Add a new example path that uses HomeField waypoints.